README.md

```diff
@@ -184,4 +189,4 @@ I would like to thank the Microverse full-stack curriculum for the inspiration a
 
```

Comment:
**[OPTIONAL BUT HIGHLY RECOMMENDED]**

- [ ] It would be nice if you make your own logo of your name and put that in the readme file at the top to present yourself as a brand in the job market or tech industry to catch the recruiters' eyes 👍